tests/Traits/InvalidCodeAnalysisTestTrait.php

```diff
@@ -16,17 +16,26 @@
 use const PHP_OS;
 use const PHP_VERSION;
 
+/**
+ * @psalm-type DeprecatedDataProviderArrayNotation = array{
+ *     code: string,
+ *     error_message: string,
+ *     ignored_issues?: list<string>,
+ *     php_version?: string
+ * }
+ * @psalm-type NamedArgumentsDataProviderArrayNotation = array{
```

Comment:
not 100% sure why, but I had issues with psalm not properly suppressing my stuff. Might be due to the fact that somewhere named arguments kicked in (due to the fact that I might've executed tests with PHP 8+ and thus had issues).
So for now, I've allowed both here, might be refactored when tests are executed on PHP 8 at some point.